### DISCUSSION TITLE
LoRaWan Certification of the Stack v4.5.1

### DISCUSSION THREAD

**Discussion by Adamzat** [Lora-net/LoRaMac-node#1031] [Q&A] [answered]
Dear All,
We are thinking the integrate the new Library supporting the LoraWan specification v1.0.4 for our products. Indeed, we would like to know if this new release has passed the certification process yet or not (for example for EU868, US915 and AS923)?
This is very important to us to decide if we can start the software development or we should wait some months until the version be stable.
Thank you for your help.
Best regards.

**Comment by mluis1** [comment, marked as answer]
Yes, all regions supported by the LoRa-Alliance certification specification have successfully run.

We have tested v4.5.0 version of this project using the LCTT tool v1.7.0 and technology package v3.1.0_R1/D1. Also, 2 test houses confirmed that all supported regions are passing all tests.

For v4.5.1 we have just run the EU868 tests as the changes shouldn't impact the different regions implementation.

I forgot to update the [releases pre certification results](https://github.com/Lora-net/LoRaMac-node/wiki/releases-pre-certification-results) to reflect the status. I'll do it as soon as possible.

In the future it would be nice if you could post this kind of questions on the project [Discussions](https://github.com/Lora-net/LoRaMac-node/discussions) tab. It is a better place to engage discussions and then we can agree if it is an issue or not.
If you don't mind I move this issue to the [Discussions](https://github.com/Lora-net/LoRaMac-node/discussions) tab.

**Reply by mluis1** [reply]
I have just updated [releases pre certification results](https://github.com/Lora-net/LoRaMac-node/wiki/releases-pre-certification-results) to reflect the current status.

**Comment by Adamzat** [comment]
Cool, Thank you :)
